src/Knp/DictionaryBundle/Form/Type/DictionaryType.php

```diff
@@ -42,14 +43,6 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
      */
     public function getParent()
     {
-        return 'choice';
-    }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function getName()
-    {
-        return 'dictionary';
+        return ChoiceType::class;
```

Comment:
Nice catch! But it seems @Shivoham changed the php minimum version to `5.5.9`: https://github.com/KnpLabs/DictionaryBundle/pull/28/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R19